/**:
  ros__parameters:
    camera_1_enabled: true
    camera_1_name: front_stereo_camera
    camera_2_enabled: false
    camera_2_name: back_stereo_camera

    labels:
      # Semantic label names defined in Isaac Sim
      names: ["unlabelled", "person"]
      
      # Definition of how the label names are converted
      # to IDs on the mask image and color on the colorized mask
      unlabelled:
        output_id: 0
        output_color: [0, 0, 0]
      person:
        output_id: 1
        output_color: [255, 0, 0] 
